### ISSUE TITLE
Exclamation mark (!) it's being ignored

### ISSUE THREAD

**Issue by tiagoporto** [semantic-release/commit-analyzer#748]
A commit `feat!: move to esm` it is being ignored and not generating a major release.

I'm using:

- `semantic-release@24.2.1`
- `pnpm@9.15.2`

```js
// release.config.mjs
export const config = {
  branches: [
    'main',
    {
      name: 'alpha',
      channel: 'alpha',
      prerelease: true,
    },
    {
      name: 'beta',
      channel: 'beta',
      prerelease: true,
    },
  ],
  preset: 'conventionalcommits',
  plugins: [
    '@semantic-release/commit-analyzer',
    '@semantic-release/release-notes-generator',
    '@anolilab/semantic-release-pnpm',
    '@semantic-release/github',
  ],
}

```

[Output](https://github.com/tiagoporto/gerador-validador-cpf/actions/runs/12769149797/job/35591077390#step:6:148)

```
[1:56:25 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  Analyzing commit: feat!: move to esm
[1:56:25 PM] [semantic-release] [@semantic-release/commit-analyzer] › ℹ  The commit should not trigger a release
```

I have test with npm, the same result from pnpm.
Also I've test adding `BREAKING CHANGE: some message` in the commit message, this works as expected. The issue is with the `!`only.

**Comment by travi** [comment]
your config file is not being loaded. you can see from the output that the `@anolilab/semantic-release-pnpm` plugin is not being loaded, but the `@semantic-release/npm` plugin is, despite it not being named in your config.

leverage the [--debug](https://semantic-release.gitbook.io/semantic-release/usage/configuration#debug) flag to make sure the config is what you expect

**Comment by tiagoporto** [comment]
Hi @travi, thanks for your help
My config file exported a `const` instead of a default export 😨.
